docs/adr/adr-009-telemetry.md

```diff
@@ -0,0 +1,220 @@
+# ADR #009: Telemetry
+
+## Changelog
+
+* 2022-07-4: Started
+* 2022-07-10: Initial Draft finished
+
+## Authors 
+
+@Wondertan @liamsi
+
+## Context
+> Now I know why I don't like writing ADRs - because I cannot run/test them and see if they work or not.
```

Comment:
😄 Actually, sometimes you can, by writing a PoC / tracer / prototype implementation of what you plan to do, or. if necessary of the different alternatives.